http/css/style.css

```diff
@@ -50,7 +50,23 @@ p { max-width: 800px; }
 
 .extension-child { padding-left: 1em; }
 .extension:hover { background-color: #babdb6; }
-.task { border-radius: 4px; padding-right: 4px; text-align: right; }
+.task {
+    position: relative;
+    border-radius: 4px;
+    padding-right: 4px;
+    height: 19px;
+    width: 72px;
+    padding: 0;
+}
+
+.task span {
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    font: 9px sans-serif;
+    text-decoration: none;
+    color: black;
+}
```

Comment:
I just want to make sure that the text doesn't scale with the user's text size preferences - the size of the cells are fixed with a pixel size.